electrum/lnwire/peer_wire.csv

```diff
@@ -180,35 +185,35 @@ msgdata,channel_update,cltv_expiry_delta,u16,
 msgdata,channel_update,htlc_minimum_msat,u64,
 msgdata,channel_update,fee_base_msat,u32,
 msgdata,channel_update,fee_proportional_millionths,u32,
-msgdata,channel_update,htlc_maximum_msat,u64,,option_channel_htlc_max
+msgdata,channel_update,htlc_maximum_msat,u64,
 msgtype,query_short_channel_ids,261,gossip_queries
 msgdata,query_short_channel_ids,chain_hash,chain_hash,
 msgdata,query_short_channel_ids,len,u16,
 msgdata,query_short_channel_ids,encoded_short_ids,byte,len
 msgdata,query_short_channel_ids,tlvs,query_short_channel_ids_tlvs,
 tlvtype,query_short_channel_ids_tlvs,query_flags,1
-tlvdata,query_short_channel_ids_tlvs,query_flags,encoding_type,u8,
+tlvdata,query_short_channel_ids_tlvs,query_flags,encoding_type,byte,
 tlvdata,query_short_channel_ids_tlvs,query_flags,encoded_query_flags,byte,...
 msgtype,reply_short_channel_ids_end,262,gossip_queries
 msgdata,reply_short_channel_ids_end,chain_hash,chain_hash,
-msgdata,reply_short_channel_ids_end,complete,byte,
+msgdata,reply_short_channel_ids_end,full_information,byte,
 msgtype,query_channel_range,263,gossip_queries
 msgdata,query_channel_range,chain_hash,chain_hash,
 msgdata,query_channel_range,first_blocknum,u32,
 msgdata,query_channel_range,number_of_blocks,u32,
 msgdata,query_channel_range,tlvs,query_channel_range_tlvs,
 tlvtype,query_channel_range_tlvs,query_option,1
-tlvdata,query_channel_range_tlvs,query_option,query_option_flags,varint,
+tlvdata,query_channel_range_tlvs,query_option,query_option_flags,bigsize,
```

Comment:
does this work?
I see that rusty renamed varint to bigsize in https://github.com/lightning/bolts/commit/9e8e29af9b9a922eb114b2c716205d0772946e56
but we have "varint" hardcoded in a few places in `lnmsg.py`, e.g.:
https://github.com/spesmilo/electrum/blob/ab953f4a7f9add8fa87c71795c36b851685b75b0/electrum/lnmsg.py#L132

Reply:
fixed in 020637b5d491a4bc7ba78017f38baf7d5a9e7f99